polylogue/importers/chatgpt.py

```
@@ -242,6 +262,8 @@ def import_chatgpt_export(
                     for conv in ijson.items(fh, "item"):
                         if not isinstance(conv, dict):
                             raise ValueError
+                        # Convert Decimal values from ijson to floats for JSON compatibility
+                        conv = _convert_decimals(conv)
```

Ruff (0.15.9)

[warning] `for` loop variable `conv` overwritten by assignment target

(PLW2901)